configure

```diff
@@ -19,6 +19,7 @@ UsageSimple() {
   echo "    -mpi           : Use mpicc/mpicxx to compile."
   echo "    -intelmpi      : Use mpiicc/mpiicpc to compile."
   echo "    -cuda          : Enable CUDA support. See SHADER_MODEL in --full-help for more info."
+  echo "    -hip           : Enable HIP support. See SHADER_MODEL in --full-help for more info."
```

Comment:
SHADER_MODEL only references CUDA -- can you update the description to include its relevance to HIP/AMD?

Reply:
Done. There are also a few more small changes, please check.